lib/pf/services/manager/haproxy.pm

```diff
@@ -196,11 +242,6 @@ sub stop {
     return $result;
 }
 
-sub isManaged {
-    my ($self) = @_;
-    return $cluster_enabled;
```

Comment:
Won't it try to bind to 127.0.0.1:3306 in the current way its generated.

That could cause issues with standalone servers that have mysql binding to 0.0.0.0:3306